.github/workflows/run_tests_CI.yml

```diff
@@ -0,0 +1,33 @@
+name: Zooni CI
+on:
+  pull_request:
+  push: { branches: master }
+env:
+  COMPOSE_FILE: docker-compose.yml
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Check for focus specs
+        run: ./scripts/no_focus.sh
+      - name: Build docker images
+        run: docker-compose build
+      - name: Create the test env DB
+        run: docker-compose run -T --rm -e RAILS_ENV=test panoptes bundle exec rake db:setup
+      - name: Setup config files
+        run: docker-compose run -T --rm -e RAILS_ENV=test panoptes bundle exec rake configure:local
+      - name: Run all the tests
+        run: docker-compose run -T --rm -e RAILS_ENV=test panoptes bundle exec rspec
+  # test:
+  #   needs: setup
+  #   runs-on: ubuntu-latest
+  #   env:
+  #     SPECS: spec/controllers/api/v1/[a-m]*.rb
+  #   steps:
+  #     - name: Run tests
+  #       run: docker-compose run -T --rm -e RAILS_ENV=test panoptes bundle exec rspec $SPECS
```

Comment:
Do we need the above commented out code?

Reply:
nope - i'll remove it.